samples/asana/program.py

```diff
@@ -0,0 +1,53 @@
+"""Demonstration of AutoKitteh's Asana integration.
+
+This script interacts with the Asana API to manage tasks, including creating new ones
+and updating existing tasks.
+"""
+
+import os
+
+from autokitteh.asana import asana_client
+
+import asana
+
+
+api_client = asana_client("asana_conn")
+client = asana.TasksApi(api_client)
+
+WORKSPACE_GID = os.getenv("WORKSPACE_GID")
```

Comment:
Ensure that the const is a string, not a string-or-None.

Also move this line above the global variables, not below them (global constant usually come before global vars).

```suggestion
WORKSPACE_GID = os.getenv("WORKSPACE_GID", "")
```